scss/jquery/overrides/_select2.scss

```diff
@@ -0,0 +1,246 @@
+// Single select2
+.select2-container {
+  // The input field overlay
+  .select2-choice {
+    background: $crm-white;
+    border-color: $crm-grayblue-darker;
+    border-radius: $border-radius-base;
+    box-shadow: $crm-form-control-shadow;
+    box-sizing: border-box;
+    height: $input-height-base;
+    line-height: 27px;
+
+    &:focus {
+      outline: 0;
+    }
+
+    // The X icon to deselect the option
+    abbr {
+      background-image: none !important;
+      right: 30px;
```

Comment:
Something similar to `15px` and `20px` concerns.

Reply:
It's the width of `.select2-arrow` + some additional margin

Reply:
created a variable